main_ww.py

```diff
@@ -70,24 +75,112 @@ def get_role(server_roles, target_id):
             await member.remove_roles(role, reason="Updating CC permissions")
         if role.id == config.suspended:
             await member.remove_roles(role, reason="Updating CC permissions")
+        if role.id == config.participant:
+            await member.remove_roles(role, reason="Updating CC permissions")
+
+already_quoted = []
+
+@client.event
+async def on_reaction_add(reaction, user):
+    if user != client.user and db_shop.is_shop(reaction.message.id):
+        # For shop
+        bought_item = await shop.find_item_from_key("emoji", reaction.emoji, reaction.message.id)
+        await reaction.message.remove_reaction(reaction.emoji, user)
+        await reaction.message.channel.send("{} just bought {} for {} {}!".format(user.mention, bought_item["name"], bought_item["price"], shop.find_shop_by_id(reaction.message.id)["currency"]))
+    elif user != client.user and reaction.emoji == "⭐":
+        # For Quoting
```

Comment:
You don't need to change it for this merge, but I'd prefer the Ghost Bot to have this functionality rather than the Werewolf Bot.